website/docs/community/spotlight/meagan-palmer.md

```diff
@@ -0,0 +1,41 @@
+---
+id: meagan-palmer
+title: Meagan Palmer
+description: |
+  I first started using dbt in 2016 or 2017 (I can't remember exactly). Since then, I have moved into data and analytics consulting and have dipped in and out of the dbt Community. 
+  Late last year, I started leading dbt Cloud training courses and spending more time in the <a href="https://www.getdbt.com/community/join-the-community/“ rel="noopener noreferrer" target="_blank">dbt Slack</a>. 
```

Comment:
i think this is what could be causing the error - should it be:  <a target="_blank" rel="noopener noreferrer" href="https://www.getdbt.com/community/join-the-community/">dbt Slack</a> instead?

Reply:
I think you're correct @mirnawong1. It looks like that fancy quote after the href is causing a parsing error.
![Spotlight_update_for_August_2024_-_images__please_merge_to_amada-spotlight-aug-2024](https://github.com/user-attachments/assets/be1e7460-0442-4b82-9517-e98a4e6720c4)